### PR TITLE
Update swagger.md

### PR DESCRIPTION
**Pull request by michaelpetri**
See https://github.com/api-platform/docs/issues/586

### CODE REVIEW

File: core/swagger.md [hunk 1]
```diff
@@ -54,6 +54,14 @@ final class SwaggerDecorator implements NormalizerInterface
 
 	// e.g. add a custom parameter
 	$docs['paths']['/foos']['get']['parameters'][] = $customDefinition;
+	
+        // e.g. remove an existing parameter
+        if ($index = $this->findParameterByName('bar', $docs['paths']['/foos']['get']['parameters'])) {
```
Comment (dunglas) [inline core/swagger.md]: Maybe can you use `array_splice` instead

Comment (dunglas) [reply core/swagger.md]: Even `array_filter`

Comment (dunglas) [reply core/swagger.md]:
```php
$docs['paths']['/foos']['get'] = array_filter($docs['paths']['/foos']['get'], function ($param) {
    return $param['name'] !== 'bar';
});
```

(not tested)

Comment (michaelpetri) [reply core/swagger.md]: Tested and also looks cleaner to me, thanks!

**Comment by michaelpetri** [comment]
@dunglas do you need additional changes?

**Comment by dunglas** [comment]
Perfect, thanks @michaelpetri!

**Pull request merged**